pages/linux/timeout.md

```diff
@@ -0,0 +1,11 @@
+# timeout
+
+> Run a command with a time limit.
+
+- Run "sleep 10" with a time limit of 3 seconds:
```

Comment:
When referring to other commands in the description texts, we normally use backticks so that they're rendered as code. Like so:

    - Run `sleep 10` with ...